Source/webserver/src/main/resources/static/js/main.js

```diff
@@ -903,4 +903,24 @@ function toggleDropDowns(div, button){
     $("img#" + button).toggleClass("change");
     $("div#" + div).toggleClass("hidden");
 }
+
+function updateUserDataDisplayMode(mediaQuery)
+{
+    if(mediaQuery.matches)
+    {
+        //Hide regular level panel, show menu level panel
+        document.getElementById('level-panel').style.display = 'none';
+        document.getElementById('menu-level-panel').style.display = 'initial';
+    }
+    else 
+    {
+        //Show regular level panel, hide menu level panel
+        document.getElementById('level-panel').style.display = 'initial';
+        document.getElementById('menu-level-panel').style.display = 'none';
+    }
+}
+
+var mediaQuery = window.matchMedia("(max-width: 600px)");
+updateUserDataDisplayMode(mediaQuery);
+mediaQuery.addListener(updateUserDataDisplayMode);
```

Comment:
This could be acheived through css. Also if the site loads slow then the both panels will show.


in Main.css there is already a media query for 600px that you can use 

add this inside it
```
main #level-panel {
      display: none;
  }

  #menu-level-panel {
      display: unset;
  }
```

And set `#main-level-panel { display: none }` in the non media-query part of the file

Reply:
![buggy_level_panel](https://user-images.githubusercontent.com/78028942/137955623-e13e4ddf-64db-417a-9221-c4ff56fe6ef4.PNG)